src/os/shared.rs

```diff
@@ -40,11 +40,17 @@ fn get_datalink_channel(
     match datalink::channel(interface, config) {
         Ok(Ethernet(_tx, rx)) => Ok(rx),
         Ok(_) => failure::bail!("Unknown interface type"),
-        Err(e) => failure::bail!(
-            "Failed to listen on network interface {}: {}",
-            interface.name,
-            e
-        ),
+        Err(e) => {
+            println!("hej");
```

Comment:
oops :)

Reply:
Haha, guess I was to fast...
Thanks for cleaning up!